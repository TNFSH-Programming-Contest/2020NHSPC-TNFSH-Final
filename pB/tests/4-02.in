85
-98.794 -669.108
-248.389 -710.173
-346.653 -705.773
-336.387 -652.975
-308.521 -696.974
-226.39 -652.975
-205.857 -516.579
-263.056 -406.583
-329.053 -305.386
-346.653 -362.584
-282.122 -472.581
-282.122 -651.508
-367.186 -578.177
-405.318 -658.842
-449.316 -554.712
-494.782 -623.643
-531.447 -581.111
-556.8 -554.8
-626.777 -560.578
-650.243 -494.58
-584.245 -475.514
-516.781 -545.912
-364.252 -471.114
-439.05 -368.451
-497.715 -386.05
-585.712 -421.249
-659.043 -322.986
-615.044 -243.788
-534.38 -295.0
-434.65 -274.587
-515.314 -88.326
-571.046 -82.46
-551.98 -224.722
-669.309 -149.924
-669.309 -66.327
-622.378 -107.392
-607.711 -23.795
-682.509 11.404
-670.776 90.601
-562.246 31.937
-575.446 108.201
-499.181 102.334
-510.914 23.137
-395.051 -119.125
-375.985 93.535
-330.52 64.202
-297.0 97.934
-176.525 117.001
-229.323 70.069
-302.654 11.404
-260.122 -6.196
-163.325 53.936
-63.595 58.336
-56.262 -64.86
-95.861 -76.593
-100.261 -0.329
-167.725 -0.329
-167.725 -51.661
-348.12 -42.861
-352.519 -179.257
-434.65 -189.523
-422.917 -214.456
-356.919 -221.789
-246.923 -312.719
-204.391 -428.582
-147.193 -459.381
-172.125 -342.052
-67.995 -350.851
-100.261 -287.787
-183.858 -221.789
-176.525 -101.526
-10.797 -98.593
-13.73 92.068
56.668 97.934
119.732 5.537
86.0 -256.988
49.335 -148.458
-119.327 -155.791
-109.06 -221.789
8.269 -211.522
64.001 -661.775
-0.53 -661.775
0.936 -399.25
-119.327 -399.25
-87.061 -512.18
-227.287 -119.202
