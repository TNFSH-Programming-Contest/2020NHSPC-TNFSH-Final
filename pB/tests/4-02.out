1183672.888